.github/workflows/builds-and-tests.yml

```diff
@@ -121,9 +121,9 @@ jobs:
       run: |
         sudo apt-get install lcov
 
-    - name: Set up Homebrew
-      if: ( matrix.mpi == 'par' || matrix.codecov == 'YES' ) && matrix.os == 'macos-latest'
-      uses: Homebrew/actions/setup-homebrew@master
+    # - name: Set up Homebrew
+    #   if: ( matrix.mpi == 'par' || matrix.codecov == 'YES' ) && matrix.os == 'macos-latest'
+    #   uses: Homebrew/actions/setup-homebrew@master
```

Comment:
Any particular reason these are commented out rather than just deleted? Do we expect to need to add them back in the future?

Reply:
I thought we may need this again for some reason -- easier to have it as a comment than trying to find it in the history or google it.